products/compute/api.yaml

```diff
@@ -2928,6 +2932,68 @@ objects:
         conflicts:
           - httpHealthCheck
+          - http2HealthCheck
+          - httpsHealthCheck
+          - tcpHealthCheck
+        properties:
+            - !ruby/object:Api::Type::String
+              name: 'request'
+              description: |
+                The application data to send once the SSL connection has been
+                established (default value is empty). If both request and response are
+                empty, the connection establishment alone will indicate health. The request
+                data can only be ASCII.
+            - !ruby/object:Api::Type::String
+              name: 'response'
+              description: |
+                The bytes to match against the beginning of the response data. If left empty
+                (the default value), any response will indicate health. The response data
+                can only be ASCII.
+            - !ruby/object:Api::Type::Integer
+              name: 'port'
+              description: |
+                The TCP port number for the SSL health check request.
+                The default value is 443.
+            - !ruby/object:Api::Type::String
+              name: 'portName'
+              description: |
+                Port name as defined in InstanceGroup#NamedPort#name. If both port and
+                port_name are defined, port takes precedence.
+            - !ruby/object:Api::Type::Enum
+              name: 'proxyHeader'
+              description: |
+                Specifies the type of proxy header to append before sending data to the
+                backend, either NONE or PROXY_V1. The default is NONE.
+              values:
+                - :NONE
+                - :PROXY_V1
+              default_value: :NONE
+            - !ruby/object:Api::Type::Enum
+              name: 'portSpecification'
+              description: |
+                Specifies how port is selected for health checking, can be one of the
+                following values:
+
+                  * `USE_FIXED_PORT`: The port number in `port` is used for health checking.
+
+                  * `USE_NAMED_PORT`: The `portName` is used for health checking.
+
+                  * `USE_SERVING_PORT`: For NetworkEndpointGroup, the port specified for each
+                  network endpoint is used for health checking. For other backends, the
+                  port or named port specified in the Backend Service is used for health
+                  checking.
+
+                If not specified, SSL health check follows behavior specified in `port` and
+                `portName` fields.
+              values:
+                - :USE_FIXED_PORT
+                - :USE_NAMED_PORT
+                - :USE_SERVING_PORT
+      - !ruby/object:Api::Type::NestedObject
+        name: 'http2HealthCheck'
+        conflicts:
+          - httpHealthCheck
+          - sslHealthCheck
           - httpsHealthCheck
           - tcpHealthCheck
         properties:
```

Comment:
`request` should be `requestPath` and its corresponding description. also a few of the other descriptions mention ssl, but should be http2